spring-ai-core/src/main/java/org/springframework/ai/vectorstore/observation/DefaultVectorStoreObservationConvention.java

```diff
@@ -24,41 +25,41 @@
 import io.micrometer.common.KeyValues;
 
 /**
+ * Default conventions to populate observations for vector store operations.
```

Comment:
The main change here is a resorting to feed Micrometer already sorted list of key values for a performance improvement, and updating the observation name to be closer to the semantic conventions